src/app/components/home/home.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Task } from './../../models/task.model';
+import { Component, OnInit } from '@angular/core';
+import { formatDate } from '@angular/common';
+import { TaskService } from 'src/app/services/task.service';
+import { UtilityService } from 'src/app/services/utility.service';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.scss'],
+
+})
+export class HomeComponent implements OnInit{
+
+    constructor(private _taskService: TaskService , private _utilityService: UtilityService){}
+    img_text = '';
+    tasks:Task[] = [];
+    task: Task = new Task();
+    show = this._utilityService.SHOW_SPLASH;
+    showAddTaskCard(){
+      this.show = true
+    }
+    // eslint-disable-next-line 
+    task_name : any
+    currentDate = formatDate(new Date(), 'dd.MM.YY', 'en');
+    
+    onAddTask(){
+      this.task_name = this.task_name.replace(/^\s+|\s+$/gm, '').trim();
+      if(this.task_name){
+        const newTask: Task = {
```

Comment:
if we're using Task as a class, let's not use it as a Type here. We can either instantiate new Task Object from the Task class and use that everywhere, or we can declare Task as an interface/Type and use it like this.

Reply:
This is create problem when we add methods to the class.

Reply:
I have created an Interface for the purpose. Thank you.